packages/@okta/vuepress-site/docs/guides/refresh-tokens/overview/index.md

```diff
@@ -2,15 +2,21 @@
 title: Overview
 ---
 
-## What is a refresh token
+Access and ID [tokens](/docs/reference/api/oidc/#tokens-and-claims) are JSON web tokens that are valid for a specific number of seconds. Typically, a user needs a new access token when they attempt to access a resource for the first time or after the previous access token that was granted to them expires.
 
-A refresh token is a special token that is used to generate additional access tokens. This allows you to have short-lived access tokens without having to collect credentials every time one expires. You request this token alongside the access and/or ID tokens as part of a user's initial authentication flow.
+A refresh token is a special token that is used to obtain additional access tokens. This allows you to have short-lived access tokens without having to collect credentials every time one expires. You request a refresh token alongside the access and/or ID tokens as part of a user's initial authentication flow. Applications must then securely store refresh tokens since they allow users to remain authenticated.
```

Comment:
Suggestion, I don't have a strong preference; `authentication flow` -> `authentication and authorization flow`